test/recipes/81-test_cmp_cli.t

```diff
@@ -64,13 +64,13 @@ my @cmp_basic_tests = (
     [ "bad int: out of range",            [ "-config", '""', "-days", "2147483648" ], 1 ],
 );
 
-my $rsp_cert = "signer_only.crt";
-my $outfile = "test.certout.pem";
-my $secret = "pass:test";
-
 # this uses the mock server directly in the cmp app, without TCP
 sub use_mock_srv_internally
 {
+    my $tmpdir = bldtop_dir("test-runs/test_cmp_cli");
+    my $secret = "pass:test";
+    my $rsp_cert = "signer_only.crt";
+    my $outfile = "$tmpdir/test.certout.pem";
```

Comment:
Actually, there is a supported command in OpenSSL::Test: `result_file`.  So you could change this line to:

``` perl
    my $outfile = result_file('test.certout.pem');
```

That way, you don't have to keep track of the exact directory yourself (i.e. you can ditch `$tmpdir`)

Reply:
Nice. I've change the test code that way,
after adapting the name of `__results_file` to `result_file` in OpenSSL::Test and exporting it.